tcms/telemetry/templates/telemetry/testing/status-matrix.html

```diff
@@ -0,0 +1,98 @@
+                    {% trans "Product" %}
+                </label>
+                <div class="col-md-3">
+                    <select name="product" id="id_product" class="form-control selectpicker">
+                        <option value="">----------</option>
+                    </select>
+                </div>
+
+                <label for="id_version" class="col-md-1 col-lg-1">
+                    {% trans "Version" %}
+                </label>
+                <div class="col-md-3">
+                    <select name="version" id="id_version" class="form-control selectpicker">
+                        <option value="">----------</option>
+                    </select>
+                </div>
+
+                <label for="id_build" class="col-md-1 col-lg-1">
+                    {% trans "Build" %}
+                </label>
+                <div class="col-md-3">
+                    <select name="build" id="id_build" class="form-control selectpicker">
+                        <option value="">----------</option>
+                    </select>
+                </div>
+            </div>
+
+            <div class="form-group">
+                <label for="id_test_plan" class="col-md-1 col-lg-1">
+                    {% trans "Test Plan" %}
+                </label>
+                <div class="col-md-3">
+                    <select name="test_plan" id="id_test_plan" class="form-control selectpicker">
+                        <option value="">----------</option>
+                    </select>
+                </div>
+
+                <label class="col-md-1 col-lg-1" for="id_after">{% trans "After" %}</label>
+                <div class="col-md-3 col-lg-3">
+                    <div class="input-group date-time-picker-pf">
+                        <input type="text" class="form-control" id="id_after">
+                        <span class="input-group-addon">
+                            <span class="fa fa-calendar"></span>
+                        </span>
+                    </div>
+
+                    {% include "datetimepicker_script.html" with selector="#id_after" %}
+                </div>
+
+                <label class="col-md-1 col-lg-1" for="id_before">{% trans "Before" %}</label>
+                <div class="col-md-3 col-lg-3">
+                    <div class="input-group date-time-picker-pf">
+                        <input type="text" class="form-control" id="id_before">
+                        <span class="input-group-addon">
+                            <span class="fa fa-calendar"></span>
+                        </span>
+                    </div>
+
+                    {% include "datetimepicker_script.html" with selector="#id_before" %}
+                </div>
+            </div>
+        </form>
+
+        <table class="table table-bordered table-with-horizontal-scroll" id="table">
+            <thead>
+            <tr>
+                <th class="header"> {% trans "Test case"%}/ {% trans "Test run" %}</th>
```

Comment:
Use `\` with equal number of spaces around it.

Reply:
Done